cypress/e2e/2270-ADO-opening-webhook-ndv-marks-workflow-as-unsaved.cy.ts

```diff
@@ -0,0 +1,20 @@
+import { NDV, WorkflowPage } from '../pages';
+
+const workflowPage = new WorkflowPage();
+const ndv = new NDV();
+
+describe('ADO-2270 Save button resets on webhook node open', () => {
+	it('should not reset the save button if webhook node is opened and closed', () => {
+		workflowPage.actions.visit();
+		workflowPage.actions.addInitialNodeToCanvas('Webhook');
+		workflowPage.getters.saveButton().click();
+		workflowPage.actions.openNode('Webhook');
+
+		ndv.actions.close();
```

Comment:
would be nice if you can use `cypress/composables/ndv` pattern here
`closeNDV()` instead for  example.
same with workflow object. but just a suggestion. 

We are trying to move away form these classes

Reply:
Good to know, thanks! Will push a change

Reply:
Ended up going with the original version as equivalent functions with `composables` are currently broken on canvas v2.